docs/guides/deployment/kubernetes.md

```diff
@@ -0,0 +1,239 @@
+
+## Prerequisites
+
+Before we begin, there are a few pre-requisites:
+
+1. A Prefect Cloud account
+2. A cloud provider (AWS, GCP, or Azure) account
+3. [Install](/getting-started/installation/) Python and Prefect
+
+!!! Note "Administrator Access"
+    Though not strictly necessary, you may want to ensure you have admin access, both in Prefect Cloud and in your cloud provider. Admin access is only necessary during the initial setup and can be downgraded after. Consult with your friendly neighborhood DevOps professional.
+
+## Create a cluster
+
+Let's start by creating a new cluster. If you already have one, skip ahead to the next section.
+
+=== "AWS"
+
+    One easy way to get set up with a cluster in EKS is with [`eksctl`](https://eksctl.io/). Node pools can be backed by either EC2 instances or FARGATE. Let's choose FARGATE so there's less to manage. The following command takes around 15 minutes and must not be interrupted:
+
+    ```bash
+    # Replace the cluster name with your own value
+    eksctl create cluster --fargate --name <CLUSTER-NAME>
+    ```
+
+<!-- === "GCP"
+    TODO
+
+=== "Azure"
+    TODO -->
+
+## Create a container registry
+
+Besides a cluster, the other critical resource we'll need is a container registry. This is also not strictly required, but in most cases you'll want to use custom images and/or have more control over where images are stored. If you already have one, skip ahead to the next section.
+
+=== "AWS"
+
+    Let's create a registry using the AWS CLI and authenticate the docker daemon to said registry:
+
+    ```bash
+    # Replace the image name with your own value
+    aws ecr create-repository --repository-name <IMAGE-NAME>
+
+    # Login to ECR
+    # Replace the region and account ID with your own values
+    aws ecr get-login-password --region <REGION> | docker login --username AWS --password-stdin <AWS_ACCOUNT_ID>.dkr.ecr.<REGION>.amazonaws.com
+    ```
+
+<!-- === "GCP"
+    TODO
+
+=== "Azure"
+    TODO -->
+
+## Create a work pool
+
+Now let's switch over to Prefect Cloud where we'll create a new work pool. Hit the plus button on the work pools page and choose Kubernetes from the list of options. On the next page set Finished Job TTL to 60 so that completed flow runs are cleaned up, and set Pod Watch Timeout Seconds to 300, especially if you are using a __serverless__ type node pool since these tend to have longer startup times. You may also want to set a custom namespace, such as `prefect`. Generally you should leave the cluster config blank as the worker will already be provisioned with appropriate access and permissions. Finally, leave the image field blank as we'll override that in each deployment.
```

Comment:
```suggestion
Now let's switch over to Prefect Cloud, where we'll create a new work pool. Hit the plus button on the work pools page and choose Kubernetes from the list of options. On the next page, set Finished Job TTL to 60 so that completed flow runs are cleaned up, and set Pod Watch Timeout Seconds to 300, especially if you are using a __serverless__ type node pool since these tend to have longer startup times. You may also want to set a custom namespace, such as `prefect`. Generally you should leave the cluster config blank as the worker will already be provisioned with appropriate access and permissions. Finally, leave the image field blank as we'll override that in each deployment.
```